src/core/manager.h

```diff
@@ -241,6 +241,8 @@ struct Manager {
         usec_t watchdog[_WATCHDOG_TYPE_MAX];
         usec_t watchdog_overridden[_WATCHDOG_TYPE_MAX];
 
+        bool runtime_watchdog_running;
```

Comment:
Can you add a brief comment here explaining why there's an additional boolean?

Reply:
Sure, done.